samples/snippets/src/main/java/com/example/dialogflow/DetectIntentTexts.java

```diff
@@ -63,7 +63,11 @@
         System.out.format(
             "Detected Intent: %s (confidence: %f)\n",
             queryResult.getIntent().getDisplayName(), queryResult.getIntentDetectionConfidence());
-        System.out.format("Fulfillment Text: '%s'\n", queryResult.getFulfillmentText());
+        System.out.format(
+            "Fulfillment Text: '%s'\n",
+            queryResult.getFulfillmentMessagesCount() > 0
+                ? queryResult.getFulfillmentMessages(0).getText()
+                : "Triggered Default Fallback Intent");
```

Comment:
nit: can we use an if/else statement here? Multi line ternaries tend to be hard to read:

Reply:
No style guide on using multiline ternary statements so this change can be ignored.